char/abstract/abstract_SKILL_script

```diff
@@ -0,0 +1,44 @@
+/*
+ Abstract Skill Generation
+ Apache 2.0 Licensing
+*/
+absSkillMode()
+absSetOption("NewLibraryName" "library")
+absSetLibrary("library")
+absSetOption("ImportLefFiles" "../techfiles/gf180mcu_5LM_1TM_9K_11t_tech.lef")
+absImportLEF()
+inp = infile("GDSlist")
+absSetOption("ImportGDSIIFiles" gets(_ inp))
+close(inp)
+absSetOption("ImportGDSIILayerMapFile" "../techfiles/stream_1P6M.map")
+absImportGDS()
+absSelectCellFrom("gf180mcu_osu_sc_12T_and2_1" "gf180mcu_osu_sc_12T_xnor2_1")
```

Comment:
curious if it would make sense to do one time conversion and commit those with `${TRACKS}` in the sources.

Reply:
Not sure what you mean here..this is a Skill script I wrote that my students use.  It just generates LEF from a gds cell regardless of tracked.